go.mod

```diff
@@ -78,7 +78,7 @@ require (
 	k8s.io/apimachinery v0.30.0
 	k8s.io/client-go v0.30.0
 	k8s.io/component-base v0.30.0
-	k8s.io/cri-api v0.30.0
+	k8s.io/cri-api v0.31.0-alpha.0.0.20240529224029-3a66d9d86654
```

Comment:
cri-ai dependency is updated with the published version
https://github.com/kubernetes/cri-api/tree/3a66d9d8665454a681e90f6c26b6173c75d5ab53

Reply:
Can we have a test for this  in CI, using cri-tools?

Reply:
Thanks, I'm now working on cri-tools PR: https://github.com/kubernetes-sigs/cri-tools/pull/1438

However, cri-tools' CI tests against released containerd/CRI-O. And, containerd's CI uses released version of cri-tools. So, there is a kind of dependency-cycle.

Thus, can we test containerd using cri-tools in a follow-up?